src/d_mode.h

```diff
@@ -56,7 +56,8 @@ typedef enum
 
 typedef enum
 {
-    exe_doom_1_2,    // Doom 1.2: shareware and registered
+    exe_doom_1_1,    // Doom 1.0/1.1: shareware and registered
```

Comment:
Are there any differences in game logic between 1.1/1.2?

I ask because this potentially affects the netgame protocol compatibility but based on the changes in the other files I don't see that there's actually anything to distinguish the two versions. Can we do without it?

Reply:
Removed in https://github.com/chocolate-doom/chocolate-doom/commit/77f49e5ea5aae45fe904306a9364018c0fc23b71 .